configure.ac

```diff
@@ -591,8 +610,8 @@ AS_CASE([$ocaml_cv_cc_vendor],
   warn_error_flag='-Werror'
   cc_warnings='-Wall -Wdeclaration-after-statement'])
 
-AS_CASE([$enable_warn_error,AC_PACKAGE_VERSION],
-  [yes,*|,*+dev*],
+AS_CASE([$enable_warn_error,OCAML_DEVELOPMENT_VERSION],
```

Comment:
This either needs to be `OCAML__DEVELOPMENT_VERSION` (two underscores) or `$OCAML_DEVELOPMENT_VERSION` (at least I think the `$` form works too - I haven't checked). The `diff` for configure shows that isn't expanding as it stands, as it's not an m4 macro.